docs/Tutorials/CrossCompilationSetup/macOS.md

```diff
@@ -1,6 +1,6 @@
-# Mac OS X
+# macOS
```

Comment:
Mac OS X is an old name for Apple's current OS see https://en.wikipedia.org/wiki/MacOS#History